pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -295,6 +295,19 @@
     @JsonIgnore
     private Clock clock = Clock.systemDefaultZone();
 
+    @ApiModelProperty(
+            name = "dnsLookupBindAddress",
+            value = "The Pulsar client dns lookup bind address."
```

Comment:
Can you add the default value and that it means the client will bind to `0.0.0.0`?

Reply:
> Can you add the default value and that it means the client will bind to `0.0.0.0`?

@michaeljmarshall the default behavior is similar, but it's not bind to `0.0.0.0`.

```java
        if (localAddress == null) {
            future = b.register();
        } else {
            future = b.bind(localAddress);
        }
```

Reply:
@michaeljmarshall I misunderstood your meaning, added the defautl behavior notion

Reply:
No worries, thanks for updating it.